base/c.jl

```diff
@@ -4,6 +4,11 @@
 
 import Core.Intrinsics: cglobal, bitcast
 
+# Equivalent to Sys.iswindows(), but that's not yet defined
+const _WINDOWS = let k = ccall(:jl_get_UNAME, Any, ())
```

Comment:
Why do we check it for `:Windows` or `:NT` in `iswindows` then? Does that mean we could simplify that conditional as well?